src/main/java/com/google/devtools/build/lib/bazel/repository/downloader/HttpStream.java

```diff
@@ -90,9 +90,6 @@ HttpStream create(
 
         if (checksum.isPresent()) {
           stream = new HashInputStream(stream, checksum.get());
-          if (retrier != null) {
-            retrier.disabled = true;
```

Comment:
The 'disabled' field is used only here and below, could you remove this field and the related code in RetryingInputStream.java?

Reply:
Sounds good to me.